brightwind/analyse/plot.py

```diff
@@ -117,19 +117,13 @@ def __init__(self):
 
             bw.analyse.plot.COLOR_PALETTE.primary_10 = '#0a1429'
 
-            # The colors used for defining the color_map can also be reset by using the example code below:
+            # The colors used for defining any of the three defined colour maps can also be reset by using the 
+            # example code below:
```

Comment:
@olivia-bentley the method you have used is not giving what expected. The reason why we had separate color map variables for each of the 3 color maps was to be able to set these separately as for examples that were given. E.g for an assessment where I want to generate plots using the different plots using the different color maps I want to be able to set my own colors and list of colors.

Reply:
I have fixed this and other bugs